tools/sync_test_server/ros-testing-server.js

```diff
@@ -37,9 +37,13 @@ function startRealmObjectServer() {
     temp.mkdir('ros', function(err, path) {
         if (!err) {
             winston.info("Starting sync server in ", path);
+            var env = Object.create( process.env );
+            winston.info(env.NODE_ENV);
+            env.NODE_ENV = 'development';
             syncServerChildProcess = spawn('realm-object-server',
                     ['--root', path,
-                    '--configuration', '/configuration.yml']);
+                    '--configuration', '/configuration.yml'],
+                    { env: env });
```

Comment:
Just curious, what is the root cause of the problem you mentioned in the sync channel -- the `make_dir() failed: No such file or directory`?